guides/example/introduction.md

```diff
@@ -17,7 +17,7 @@ The app demonstrates how to:
 
 - Build dashboard pages using:
   - [Wix Design System](https://www.wixdesignsystem.com/) React components
-  - [Wix Dashboard SDK](https://dev.wix.com/docs/client/api-reference/dashboard-sdk/intro)
+  - [Wix Dashboard React SDK](https://dev.wix.com/docs/client/api-reference/dashboard-sdk/intro)
```

Comment:
Need to update the link too right?
<img width="783" alt="image" src="https://github.com/wix-incubator/wix-cli-docs/assets/2660550/78c4d599-bf21-44b4-bd04-2409ef513eb2">

Reply:
yep, once the dashboard react docs are published
https://github.com/wix-private/developer-docs/pull/1573
waiting on a tech writer review